src/v/cloud_storage/materialized_resources.cc

```diff
@@ -144,9 +144,8 @@ get_throughput_limit(std::optional<size_t> device_throughput) {
     }
 
     auto tp = std::min(hard_limit, device_throughput.value());
-    constexpr auto tput_overshoot_frac = 16;
     return {
-      .disk_node_throughput_limit = tp + (tp / tput_overshoot_frac),
+      .disk_node_throughput_limit = tp,
```

Comment:
> Revert "cloud_storage: Increse disk tput limit a bit"
This reverts commit https://github.com/redpanda-data/redpanda/commit/0ae2ad2c7ba6f56250b74ffe2c403557fbce8988.

Needs explanation

Reply:
that was previous attempt to fix this test by increasing the disk limit a bit to make sure that the net limit is used to throttle downloads

Reply:
@nvartolomei please make sure that you put context into the commit messages. not having to visit github to see that context is a powerful thing.

Reply:
🤦 

I did fix the commits as you asked:

* https://github.com/redpanda-data/redpanda/commit/b9fcc7a7d2db43566c5207b15c005d6ebd0e67be
* https://github.com/redpanda-data/redpanda/commit/6f213fbf0d75268944d54c8a7dba5094dee266ad

But then I did a **force push from a different directory** to rebase on dev 🤦🤦🤦